src/plugins/vis_type_vislib/public/heatmap.ts

```diff
@@ -48,6 +49,9 @@ export const createHeatmapVisTypeDefinition = (deps: VisTypeVislibDependencies)
   description: i18n.translate('visTypeVislib.heatmap.heatmapDescription', {
     defaultMessage: 'Shade cells within a matrix',
   }),
+  getSupportedTriggers: () => {
+    return [VIS_EVENT_TO_TRIGGER.filter, VIS_EVENT_TO_TRIGGER.brush];
```

Comment:
🤔 Does hit map support brush? Quickly pocked into it and didn't trigger brush

Reply:
I was a little confused with it as it has on its config
`events: {
    brush: { disabled: false },
  },`

I will remove it! Thanx for that.

Reply:
I am not 100% sure! but I guess this is the case..